docs/manual/troubleshooting.tex

```diff
@@ -825,11 +825,12 @@
 \begin{enumerate}
 
 \item
-Run \code{./gradlew assemble} to build the Checker Framework:
+Run \code{./gradlew copyJarsToDist} to build the Checker Framework (ignore
```

Comment:
This section shows how to build the Checker Framework for use the Javac wrapper.  I think that should be made clear.  Then perhaps also explain how to publish artifacts to Maven local.

This pull request is related to https://github.com/typetools/checker-framework/pull/5563. I do think this is an improvement, but I think we should add a task that has a better name.  (See comments on https://github.com/typetools/checker-framework/pull/5563).

Reply:
Rename to `assembleForJavac`